inc/options/install.php

```diff
@@ -1,100 +1,422 @@
+					),
+					'acronym' => array(
+						'title' => array(),
+					),
+				)
+			);
+			?>
+		</li>
+		<li>
+			<?php
+			echo wp_kses(
+				sprintf(
+					// translators: 1 opening HTML em tag, 2 closing HTML em tag,
+					// translators: 3 opening HTML acronym tag, 4 closing HTML acronym tag
+					// translators: 5 opening HTML a tag to W3TC CDN admin page, 6 closing HTML a tag,
+					// translators: 7 opening HTML acronym tag, 8 closing HTML acronym tag,
+					// translators: 9 opening HTML acronym tag, 10 closing HTML acronym tag,
+					// translators: 11 opening HTML acronym tag, 12 closing HTML acronym tag,
+					// translators: 13 opening HTML acronym tag, 14 closing HTML acronym tag.
+					__(
+						'%1$sRecommended:%2$s If you already have a content delivery network (%3$sCDN%4$s) provider, proceed to the "%5$sContent Delivery Network%6$s" tab and populate the fields and set your preferences. If you do not use the Media Library, you will need to import your images etc into the default locations. Use the Media Library Import Tool on the "Content Delivery Network" tab to perform this task. If you do not have a %7$sCDN%8$s provider, you can still improve your site\'s performance using the "Self-hosted" method. On your own server, create a subdomain and matching %9$sDNS%10$s Zone record; e.g. static.domain.com and configure %11$sFTP%12$s options on the "Content Delivery Network" tab accordingly. Be sure to %13$sFTP%14$s upload the appropriate files, using the available upload buttons.',
+						'w3-total-cache'
+					),
+					'<em>',
+					'</em>',
+					'<acronym title="' . __( 'Content Delivery Network', 'w3-total-cache' ) . '">',
+					'</acronym>',
+					'<a href="admin.php?page=w3tc_cdn">',
+					'</a>',
+					'<acronym title="' . __( 'Content Delivery Network', 'w3-total-cache' ) . '">',
+					'</acronym>',
+					'<acronym title="' . __( 'Domain Name System', 'w3-total-cache' ) . '">',
+					'</acronym>',
+					'<acronym title="' . __( 'File Transfer Protocol', 'w3-total-cache' ) . '">',
+					'</acronym>',
+					'<acronym title="' . __( 'File Transfer Protocol', 'w3-total-cache' ) . '">',
+					'</acronym>'
+				),
+				array(
+					'a'       => array(
+						'herf' => array(),
+					),
+					'em'      => array(),
+					'acronym' => array(
+						'title' => array(),
+					),
+				)
+			);
+			?>
+		</li>
+		<li>
+			<?php
+			echo wp_kses(
+				sprintf(
+					// translators: 1 opening HTML em tag, 2 closing HTML em tag,
+					// translators: 3 opening HTML a tag to W3TC DatabaseCache admin page, 4 closing HTML a tag.
+					__(
+						'%1$sOptional:%2$s On the "%3$sDatabase Cache%4$s" tab the recommended settings are preset. If using a shared hosting account use the "disk" method with caution; in either of these cases the response time of the disk may not be fast enough, so this option is disabled by default.',
+						'w3-total-cache'
+					),
+					'<em>',
+					'</em>',
+					'<a href="admin.php?page=w3tc_dbcache">',
```

Comment:
Use `esc_url()` and `admin_url()` for the `href`.

Reply:
Added esc_url and admin_url to href